gravelsieve/init.lua

```diff
@@ -52,15 +52,17 @@ local settings_get
 if minetest.setting_get then
 	settings_get = minetest.setting_get
 else
-	settings_get = function(...) endminetest.settings:get(...) end
+	settings_get = function(...) return minetest.settings:get(...) end
```

Comment:
Forgive me if I'm wrong but this was very broken before right? It's only a fallback, but still...